recipes/dynare_preprocessor/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: dynare_preprocessor
+  version: 0.0.1
+
+source:
+  url: https://git.dynare.org/Dynare/preprocessor/-/archive/master/preprocessor-master.tar.gz
+  sha256: 25e1b0f30d49b24daf520223e1d1919e9a93dae32d6156f8269014ef2f5aa85e
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
+    - git-lfs
+    - meson
+    - gsl
+    - openlibm
+    - boost-cpp
+    - boost
+    - pkgconfig
```

Comment:
```suggestion
    - pkg-config
```

Reply:
Thank you @xhochy !
I followed exactly your suggestions, but made separate commits instead of using the github interface. Sorry for that. Can you mark the issues as resolved ?